config/Config-kernel.in

```diff
@@ -1166,6 +1172,13 @@ if KERNEL_IPV6
 	config KERNEL_LWTUNNEL_BPF
 		def_bool n
 
+        config KERNEL_MPTCP_IPV6
+                bool "Enable IPv6 MultiPath TCP support"
+                default n
```

Comment:
This can default to y as it already depends on KERNEL_MPTCP. I believe most people enabling KERNEL_MPTCP will prefer ipv6 support included :)